integration-tests/test/privateDataExchange.ts

```diff
@@ -27,21 +27,15 @@ const privateFactKey = 'privatedata_fact';
 let stakeWei = new BN('100000', 10);
 const contractCreationParams: any = {};
 
-if (getNetwork() === NetworkType.Quorum) {
+const isQuorum = getNetwork() === NetworkType.Quorum;
+if (isQuorum) {
   // On quorum we use accounts which does not have money, so stake 0. However, on ganache - each account has eth
   stakeWei = new BN('0', 10);
-}
-
-if (isPrivateTxMode) {
-  switch (getNetwork()) {
-    case NetworkType.Quorum:
-      contractCreationParams.privateFor = [getNodePublicKey(1)];
-      options = {
-        txRetriever: ext.quorum.getPrivateTx,
-      };
-      break;
-    default:
-      break;
+  if (isPrivateTxMode) {
```

Comment:
This change is not correct. `switch` statement was left in mind so we could easily add another network case which supports private transactions. Please revert this